### ISSUE TITLE
Training crashes when reaches a plateau

### ISSUE THREAD

**Issue by DomInvivo** [datamol-io/graphium#32]
When running the configurations `config_ZINC_bench_gnn.yaml`, after 32 epochs, a plateau is reached. However, instead of reducing the learning rate, it starts to only train on the first batch of each epoch and ignores the rest, until it reaches the minimum number of epochs required to stop.
